src/data/models/User.ts

```diff
@@ -7,25 +7,38 @@
  * LICENSE.txt file in the root directory of this source tree.
  */
 
+/**
+ * TODO(Mike): Remove initializers once this bug is fixed
+ * https://github.com/sequelize/sequelize/issues/10579
+ */
+
```

Comment:
Nit: Add TODO to resolve issue with child tables as well. ie: UserProfile being null.